### PR TITLE
Added invoke workflow to the workflow client

### PR DESCRIPTION
**Pull request by JeffreyThiessen**
run workflow is a deprecated call in bioblend, so this PR adds the updated invoke workflow call.

### CODE REVIEW

**Comment by JeffreyThiessen** [comment]
Also added `importWorkflow(String json, boolean publish)` to match bioblend's call structure for `import_workflow_dict`. It behaves in the same way but also sends the boolean for publish. The previous `import_workflow_json` which matches `importWorkflow(String json)` is deprecated in the galaxy api.

File: src/main/java/com/github/jmchilton/blend4j/galaxy/WorkflowsClientImpl.java [hunk 1]
```diff
@@ -42,13 +44,25 @@ public WorkflowOutputs runWorkflow(final WorkflowInputs workflowInputs) {
     return runWorkflowResponse(workflowInputs).getEntity(WorkflowOutputs.class);
   }
 
-  public ClientResponse importWorkflowResponse(final String json) {
-    final String payload = String.format("{\"workflow\": %s}", json);
+  public ClientResponse runWorkflowInvocationResponse(WorkflowInvocationInputs workflowInvocationInputs) {
+    return create(getWebResource().path(workflowInvocationInputs.getWorkflowId()).path("invocations"),workflowInvocationInputs);
+  }
+
+  public WorkflowInvocationOutputs invokeWorkflow(final WorkflowInvocationInputs workflowInvocationInputs) {
```
Comment (apetkau) [inline src/main/java/com/github/jmchilton/blend4j/galaxy/WorkflowsClientImpl.java]: Thanks so much @JeffreyThiessen. Overall this looks great.

One comment I have for this class, could you add a `@Deprecated` annotation to the `runWorkflow()` and `runWorkflowResponse()` methods?

Comment (JeffreyThiessen) [reply src/main/java/com/github/jmchilton/blend4j/galaxy/WorkflowsClientImpl.java]: Good catch. Added.

**Comment by apetkau** [comment]
This looks great. Thanks @JeffreyThiessen. Sorry for the delay in reviewing it.

**Pull request merged**